### PR TITLE
Rearange long enter menu

### PR DESCRIPTION
**Pull request by 3djc**
New order :
Model Select
Monitors
Reset
Statistics
About

### CODE REVIEW

File: radio/src/gui/horus/view_main.cpp [hunk 1]
```diff
@@ -29,6 +29,8 @@
 #define TRIM_LEN                       80
 #define POTS_LINE_Y                    (LCD_H-20)
 
+extern uint8_t lastMonitorPage;
```
Comment (bsongis) [inline radio/src/gui/horus/view_main.cpp]: It should be inside menus.h then, I think there is another location it could be removed

Comment (3djc) [inline radio/src/gui/horus/view_main.cpp]: Yeap, doing it now

Comment (3djc) [inline radio/src/gui/horus/view_main.cpp]: It hum *was* already ion there, just forgot it was :D

**Comment by bsongis** [comment]
All good, thanks!

**Pull request merged**